falcon/response_helpers.py

```diff
@@ -85,14 +85,14 @@ def format_range(value):
     return result
 
 
-def format_content_disposition(value):
+def format_content_disposition(value, disposition_type='attachment'):
```

Comment:
I was torn about whether to add a `disposition_type: str` arg like this, or `inline: bool` and drop a
```
disposition_type = 'inline' if inline else 'attachment'
```
at the start of the function. Thoughts? If I keep it this way, should there be any sort of validation on the value of `disposition_type`? Say, to ensure there are no newlines at least?

Reply:
I think the current version is ok, better than `inline:bool` personally, since if there is new disposition in the future, say `'tophone'`, we can just reuse the function with deprecation/etc.

Regarding validation, I think it's fine as is, since this is not really a public facing api, and even if called it just returns a string. You can't prevent an user from shooting themself in the foot :)